activerecord/lib/active_record/inheritance.rb

```diff
@@ -37,6 +37,7 @@ module Inheritance
 
     included do
       # Determines whether to store the full constant name including namespace when using STI.
+      # This is true, by default.
```

Comment:
no, it's okay, you can leave it. I'm seeing this in other files too, so it's okay :)